build.sc

```diff
@@ -594,7 +594,7 @@ def launcherScript(shellJvmArgs: Seq[String],
          |# Client-server mode doesn't seem to work on WSL, just disable it for now
          |# https://stackoverflow.com/a/43618657/871202
          |if grep -qEi "(Microsoft|WSL)" /proc/version > /dev/null 2> /dev/null ; then
-         |    ${java("mill.MillMain")}
+         |    COURSIER_CACHE=out/.coursier ${java("mill.MillMain")}
```

Comment:
So we use a local coursier cache under `out` on Windows but not under the other OSes. Why it that. Shouldn't we use a location that it less sensitive to `mill clean`?

Reply:
I use a local coursier cache under *Windows Subsystem for Linux*, now just Windows! I do this because under WSL, your code is running in a VM, and so the home folder that coursier normally puts stuff in is inaccessible from the host machine that IntelliJ is running on.

We could pick another location; I've considered just `.coursier/`. It definitely is something you don't want to clean accidentally due to the time taken to download